### ISSUE TITLE
USWDS-Site - Bug: Content does not meet color contrast minimum

### ISSUE THREAD

**Issue by alex-hull** [uswds/uswds-site#2829]
### Describe the bug

_**Note**_: This appears to be a false negative at least for the web standards button. It is reading the background as white versus the dark blue. Other tests are just slightly off from passing. It may be a matter of increasing the font by a single point.

Siteimprove is flagging that content on pages does not meet minimum contrast standards. This is bringing the score down .03 points. After going through the issues, it appears it is flagging three main things: the red and gold subheadings on the [Transforming the American digital experience](https://designsystem.digital.gov/next/) pages, and the "Read website standards" button at the top of each page. This is being flagged on 207 different pages. 

### Steps to reproduce the bug

1. Take the hex codes of each color
2. Run the codes through a color contrast checker
3. Check that it meets a contrast of 4.5:1, in accordance to WCAG standards

### Expected Behavior

The contrast reader will say that our ratio is above a 4.5:1 ratio

### Screenshots

<img width="223" alt="Screenshot 2024-09-13 115626" src="https://github.com/user-attachments/assets/c3a37052-1ffe-46de-bdc5-fcf49551cb3d">
<img width="225" alt="Screenshot 2024-09-13 154438" src="https://github.com/user-attachments/assets/3f09d986-4bb0-4777-becc-5ea45550e818">
<img width="224" alt="Screenshot 2024-09-13 154531" src="https://github.com/user-attachments/assets/eecf0d43-5f11-4ff8-ac2f-1e4948a06262">

### Additional context

Siteimprove and CCA are reading the background on the Transforming the American digital page as off white, and you will see that as such. 

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds-site/issues) for duplicate bug reports.

**Comment by amycole501** [comment]
Noting that ANDI finds the same issue

![Image](https://github.com/user-attachments/assets/5b623b99-6b16-4a39-a345-6d7f0231b3d5)
